dashboard/lib/projects_list.rb

```diff
@@ -0,0 +1,41 @@
+module ProjectsList
+  class << self
+    def fetch_section_projects(section)
```

Comment:
Please document this public function, ideally in [YARD](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md#Which_Markup_Format_) syntax, especially since in `lib` it may not be entirely obvious what section is.